src/ConnectionRequest.jl

```diff
@@ -82,6 +82,9 @@ function request(::Type{ConnectionPoolLayer{Next}}, url::URI, req, body;
 
     try
         if proxy !== nothing && target_url.scheme == "https"
+            target_url = URI(scheme=target_url.scheme,
```

Comment:
I think we could just do `target_url = merge(target_url, port=443)`, right? That way we keep the rest of the uri components?

Reply:
Yes, that is better.
I have made the change and successfully executed `HTTP.request("GET", "https://httpbin.org/ip")`.